Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers_test.py

```diff
@@ -243,3 +243,29 @@ def test_update_query(query, filter_name, filter_value, filter_operator, expecte
     from AzureRiskyUsers import update_query
     query = update_query(query, filter_name, filter_value, filter_operator)
     assert query == expected_query
+
+
+def test_do_pagination(mocker):
```

Comment:
add more test cases

1. you have next link
2. you dont have next link
3. you have page size

etc